pkg/grizzly/grafana.go

```diff
@@ -191,13 +230,18 @@ func getDashboard(config Config, uid string) (*Board, error) {
 
 	type nestedBoard struct {
 		Dashboard Board `json:"dashboard"`
+		Meta      struct {
+			FolderID    int64  `json:"folderId"`
+			FolderTitle string `json:"folderTitle"`
+		} `json:"meta"`
 	}
+
```

Comment:
Can you drop a comment here on why it is nested? (I think I introduced this but apparently it is less clear as I thought it would be).

Reply:
that's just how the JSON comes back from the API.